google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -1795,25 +1830,32 @@ public void testWriteLogEntriesWithSeverityFlushEnabled() {
 
   @Test
   public void testWriteLogEntriesWithOptions() {
-    Map<String, String> labels = ImmutableMap.of("key", "value");
-    WriteLogEntriesRequest request =
-        WriteLogEntriesRequest.newBuilder()
-            .putAllLabels(labels)
-            .setLogName(LOG_NAME_PB)
-            .setResource(MONITORED_RESOURCE.toPb())
-            .addAllEntries(
-                Iterables.transform(
-                    ImmutableList.of(LOG_ENTRY1, LOG_ENTRY2), LogEntry.toPbFunction(PROJECT)))
-            .build();
-    WriteLogEntriesResponse response = WriteLogEntriesResponse.newBuilder().build();
-    EasyMock.expect(loggingRpcMock.write(request)).andReturn(ApiFutures.immediateFuture(response));
-    EasyMock.replay(rpcFactoryMock, loggingRpcMock);
-    logging = options.getService();
-    logging.write(
-        ImmutableList.of(LOG_ENTRY1, LOG_ENTRY2),
-        WriteOption.logName(LOG_NAME),
-        WriteOption.resource(MONITORED_RESOURCE),
-        WriteOption.labels(labels));
+    test_write_log_entries_with_destination(
+        PROJECT, LOG_NAME_PB, LogDestinationName.project(PROJECT));
+  }
+
+  @Test
+  public void testWriteLogEntriesWithProjectOverrideOptions() {
+    test_write_log_entries_with_destination(
+        PROJECT, LOG_NAME_OVERRIDE_PB, LogDestinationName.project(PROJECT_OVERRIDE));
+  }
```

Comment:
what is the difference between this test and the `testWriteLogEntriesWithOptions` test beside different values of the project id in the parameters?
It looks like the same test to me.

Reply:
It checks cases when project ID used in `LogEntry.toPbFunction()` is same as for request and case when it is different.